corehq/apps/accounting/management/commands/suppress_duplicate_invoices.py

```diff
@@ -0,0 +1,97 @@
+from django.core.management.base import BaseCommand
+from django.db.models import Count
+from corehq.apps.accounting.interface import get_subtotal_and_deduction
+from corehq.apps.accounting.models import Invoice, CreditLine, FeatureType
+from corehq.util.dates import get_first_last_days
+
+
+class Command(BaseCommand):
+    help = 'Put credit back for duplicate invoices and suppress them'
+
+    def add_arguments(self, parser):
+        parser.add_argument('year', type=int, help='The year of the statement period.')
+        parser.add_argument('month', type=int, help='The month of the statement period.')
+        parser.add_argument('note', type=str,
+                            help='A note to be added when crediting back the amount')
+        parser.add_argument('--dry-run', action='store_true', default=False)
+
+    def handle(self, *args, **kwargs):
+        year = kwargs['year']
+        month = kwargs['month']
+        note = kwargs['note']
+        dry_run = kwargs['dry_run']
+        dry_run_tag = '[DRY_RUN]' if dry_run else ''
+
+        start_date, end_date = get_first_last_days(year, month)
+
+        # Filter the Invoice objects based on the date_start
+        invoices_of_given_date = Invoice.objects.filter(date_start__range=(start_date, end_date),
+                                                        is_hidden_to_ops=False)
+
+        # Extract the subscription ids that have duplicate invoices
+        duplicate_subs_ids = invoices_of_given_date.values('subscription').annotate(
+            count=Count('id')).filter(count__gt=1).values_list('subscription', flat=True)
+
+        suppressed_count = 0
+        for sub_id in duplicate_subs_ids:
+            related_invoices = list(Invoice.objects.filter(
+                subscription_id=sub_id, date_start__range=(start_date, end_date)).order_by('-date_created'))
```

Comment:
I think we also need the filter `is_hidden_to_ops=False` here as well

Reply:
Ah yes! Fixed here: ff3d7a571ad7ebb54d6d35185ed7c19734137dda